config/connectMongoDB.js

```diff
@@ -0,0 +1,18 @@
+const mongoose = require("mongoose");
+
+const connectMongoDB = () => {
+  mongoose.connect(process.env.MONGO_DB,
+    {
+      useNewUrlParser: true,
```

Comment:
```suggestion
  mongoose.connect(process.env.MONGO_DB, {
    useNewUrlParser: true,
```
이렇게 들어가는 게 자연스럽지 않을까요..?